### PR TITLE
fix: adding placeholder color and adding rtl to the button

### PR DESCRIPTION
**Pull request by Lokanathan-k**
## Related Issue
part of: https://github.com/SAP/fundamental-ngx/issues/5025

## Description
adding placeholder color for text area and adding rtl to the button
 

### Before:
Screenshot 2021-07-13 at 6 10 11 PM

### After:
Screenshot 2021-07-13 at 6 10 34 PM

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [x] All values are in `rem`
- [x] Text elements follow the truncation rules
- [x] hover state of the element follow design spec
- [x] focus state of the element follow design spec
- [x] active state of the element follow design spec
- [x] selected state of the element follow design spec
- [x] selected hover state of the element follow design spec
- [x] pressed state of the element follow design spec
- [x] Responsiveness rules - the component has modifier classes for all breakpoints
- [x] Includes Compact/Cosy/Tablet design
- [x] RTL support
2. The code follows fundamental-styles code standards and style
- [x] only one top level `fd-*` class is used in the file
- [x] BEM naming convention is used
- [x] Mixins are used for repeatable code (`fd-rtl`, `fd-ellipsis`, `fd-flex`, `fd-selected`, `fd-focus`, ect.)
- [x] A11y support - keyboard support, screenreader support, proper ARIA attributes, etc.
- [x] `fd-reset()` mixin is applied to all elements
- [x] Variables are used, if some value is used more than twice.
- [x] Checked if current components can be reused, instead of having new code.
3. Testing
- [x] tested Storybook examples with "CSS Resources" `normalize` option 
- [x] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [x] Verified all styles in IE11
- [x] Updated tests
- [x] last commit message should have `[ci visual]` so it can trigger chromatic visual regression (e.g. `test: run chromatic visual regression [ci visual]`)
4. Documentation
- [NA] Storybook documentation has been created/updated
- [x] Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

File: src/feed-input.scss [hunk 1]
```diff
@@ -49,6 +51,35 @@ $block: #{$fd-namespace}-feed-input;
...
+
+      &:-ms-input-placeholder {
+        color: #32363a;
+        color: var(--sapTextColor, #32363a);
+      }
+
+      &:-moz-placeholder {
+        color: #32363a;
+        color: var(--sapTextColor, #32363a);
+      }
```
Comment (InnaAtanasova) [inline src/feed-input.scss:75]: - you don't need the fallback

File: src/feed-input.scss [hunk 2]
```diff
@@ -49,6 +51,35 @@ $block: #{$fd-namespace}-feed-input;
...
+    }
+  }
+
+  .fd-button {
+    [dir="rtl"] &,
+    &[dir="rtl"] {
```
Comment (InnaAtanasova) [inline src/feed-input.scss:81]: There's a mixin for this

File: src/feed-input.scss [hunk 3]
```diff
@@ -27,6 +27,8 @@ $block: #{$fd-namespace}-feed-input;
 
   @include fd-disabled() {
     cursor: not-allowed;
+    pointer-events: initial;
+    opacity: initial;
```
Comment (InnaAtanasova) [inline src/feed-input.scss:31]: why?

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]

File: src/feed-input.scss (hunk 1 above)
Comment (InnaAtanasova) [reply src/feed-input.scss:75]: Do you really need `&::-webkit-input-placeholder`, `&::-moz-placeholder`, etc.? Have you tested without them?

File: src/feed-input.scss (hunk 3 above)
Comment (Lokanathan-k) [reply src/feed-input.scss:31]: Without that the screen looks like this in NGX which is not seen in the style
Screenshot 2021-07-14 at 5 16 39 PM

File: src/feed-input.scss (hunk 2 above)
Comment (Lokanathan-k) [reply src/feed-input.scss]: fixed

File: src/feed-input.scss (hunk 1 above)
Comment (Lokanathan-k) [reply src/feed-input.scss]: Fixed, removed since its working in the latest version of firefox and other browser

File: src/feed-input.scss [hunk 4]
```diff
@@ -27,6 +27,15 @@ $block: #{$fd-namespace}-feed-input;
...
+
+    textarea {
```
Comment (InnaAtanasova) [inline src/feed-input.scss:33]: `textarea::placeholder { ... }`

File: src/feed-input.scss [hunk 5]
```diff
@@ -46,9 +55,11 @@ $block: #{$fd-namespace}-feed-input;
     max-height: $fd-feed-input-max-height;
     margin: 0;
     padding: 0.3125rem 0.5rem;
+  }
 
-    &::placeholder {
-      font-style: var(--fdFeed_Input_Placeholder_Font_Style);
+  .fd-button {
```
Comment (InnaAtanasova) [inline src/feed-input.scss:60]: Can't do this

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]

**Review by InnaAtanasova — APPROVED** [review]

File: src/feed-input.scss (hunk 3 above)
Comment (kavya-b) [reply src/feed-input.scss:31]: will `initial` work in IE, have you checked? https://caniuse.com/?search=initial says no support for IE

**Pull request merged**